server/controllers/shopController.js

```diff
@@ -4,7 +4,22 @@ const pool = require('../../db/index');
 exports.exampleGet = async (req, res) => {
   // SAMPLE QUERY
   const query = `
-    SELECT * FROM products LIMIT 10
+    SELECT * FROM products WHERE categories = 'Meals'
```

Comment:
A get request to `/products/` currently will get only meals with this update. Should we split this between `/shop/products/` getting the individual items & `/shop/meals/` getting the meals instead? Just a suggestion.

Reply:
Yep - in the end we will have two separate requests; I will add this on the next commit, for now this should work okay.